lib/temporal/activity/task_processor.rb

```diff
@@ -56,20 +57,20 @@ def queue_time_ms
         ((started - scheduled) * 1_000).round
       end
 
-      def respond_completed(result)
-        Temporal.logger.info("Activity #{activity_name} completed")
+      def respond_completed(result, context)
```

Comment:
I'm assuming the intention was to pass `metadata`, not `context`. You also don't need to do this since metadata is available via the `attr_reader`

Reply:
Oops, left over from a previous revision.